src/strings/en-us.json

```
@@ -1565,5 +1565,11 @@
     "ButtonCast": "Cast",
     "ButtonPlayer": "Player",
     "StopPlayback": "Stop playback",
-    "ClearQueue": "Clear queue"
+    "ClearQueue": "Clear queue",
+    "HeaderSelectFallbackFontPath" : "Select Fallback Font Folder Path",
+    "HeaderSelectFallbackFontPathHelp": "Browse or enter the path of the fallback font folder to use for rendering ASS/SSA subtitles.",
+    "LabelFallbackFontPath": "Fallback font folder path:",
+    "LabelFallbackFontPathHelp": "Specify a path containing fallback font for rendering ASS/SSA subtitles. The maximum allowed total font size is 20 Megabytes. It is recommended to use the lightweight and web-friendly woff2 format font. Leave it blank unless the font is rendered incorrectly.",
```

If fallback fonts are enabled with "EnableFallbackFont", there should be no reason to leave this blank (if you're leaving it blank, you should probably also disable the option entirely unless there are plans to have default _and configurable_ fallback fonts); maybe remove that guidance from the help text? See suggested change.

Restructured the help text for brevity and noun plurality. See suggested change.

jellyfin/jellyfin#3683 has the maximum individual size set to 10 MB and the server-side logging announces this as well. Please consider adding this to the help text alongside the max limit or remove the limit altogether (and let the server administrators decide what is best for their individual situation) per the suggestion from Bond-009: ["Why even have a max size?"](https://github.com/jellyfin/jellyfin/pull/3683#issuecomment-663647706) I'll leave this decision to others.

```suggestion
    "LabelFallbackFontPathHelp": "Specify a path containing fallback fonts for rendering ASS/SSA subtitles. The maximum allowed total font size is 20 MB. Lightweight and web-friendly font formats such as woff2 are recommended.",
```

---

You may want to include all 4 supported formats here as you did in the body of the PR. That might be too verbose though; I'll leave that up to you and the other reviewers.

---

> If fallback fonts are enabled with "EnableFallbackFont", there should be no reason to leave this blank (if you're leaving it blank, you should probably also disable the option entirely unless there are plans to have default and configurable fallback fonts)

There is currently no plan to preset fallback fonts on the server. Now only when the path is not empty can the fallback font be enabled. Otherwise the box will be automatically unchecked.

> jellyfin/jellyfin#3683 has the maximum individual size set to 10 MB and the server-side logging announces this as well. Please consider adding this to the help text alongside the max limit or remove the limit altogether

Now the total size of all fonts is 20 MB as the upper limit. This is sufficient for rendering subtitles using CJK subset fonts. Large fonts are very unfriendly when bandwidth is limited. But the end user may not know this.